docs/Screenshots.stories.mdx

```diff
@@ -0,0 +1,55 @@
+<Meta
+  title="Guidelines/Screenshot testing"
+  parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+## Screenshot testing
+
+For the starting work with the screenshot test locally you need to start `storybook`:
+
```

Comment:
suggestion:
`In order to start working with screenshot testing locally, you need to start Storybook.`